docs/getting-started.md

```diff
@@ -105,3 +110,35 @@ Or to start the auto-restarting development server:
 ```sh
 make start-dev
 ```
+
+Run Docker for development
+----------------------
+
+The `Dockerfile.dev` and `docker-compose.yml` file at the root of the project work together to start the
+development server with a MySQL database.
+
+Note: so far this has been built with a bias for ARM64/Apple Silicon machines. 
+
+Prerequisites:
+
+- [Docker Desktop](https://docs.docker.com/desktop/) or CLI
+
+### Setup
+1. Copy the `.env.defaults` file to `.env` and edit the variables to your liking. 
+1. Ensure the variables match the values in the `docker-compose.yml` file.
+1. Create a RSA JWT private key at the root of the project with `openssl genrsa -out ./jwt_secret.key 4096`
+It will be read by the server from as the `JWT_PRIVATE_KEY` environment variable.
```

Comment:
Sadly this will not work. `JWT_PRIVATE_KEY` has to be a full private key, not just a path to a private key.

Probably the best way to do this is using the `secrets` feature:

https://docs.docker.com/compose/how-tos/use-secrets/

Reply:
ah, I had that as a separate section at one point 🤪